book/src/usage.md

```diff
@@ -10,32 +10,143 @@
 - [Moving the selection with syntax-aware motions](#moving-the-selection-with-syntax-aware-motions)
 <!--toc:end-->
 
-For a full interactive introduction to Helix, refer to the
-[tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) which
-can be accessed via the command `hx --tutor` or `:tutor`.
 
 > 💡 Currently, not all functionality is fully documented, please refer to the
 > [key mappings](./keymap.md) list.
 
-## Registers
+## Opening a file
+
+When you start Helix (`hx`), it automatically opens a file-picker. You can return to file-picker anytime
+with `<space>f`. To change config of the file you will need to enter insert mode with `i` to perform a
+change and leave it with `<esc>`. To add a newline below and enter insert mode `o`. For a newline above
+your cursor it's `O`.
+
+> 💡 If you are a new user, cursor navigation, backspace and delete are fully supported, however
+> it's recommended to learn j,k,h,l for navigation. This allows you to keep your fingers on your keyboard
+> ergonomically and gradually you will start using more efficient ways to navigate and change your text,
+> rather repeat-pressing arrow keys. Vim/Helix users call this "modal editing" paradigm
+
+If you perform a change by mistake use `u` to undo and `U` to re-do.
+
+## `:` Menu
+
+Opening a menu provides access to the most essential functions of Helix such as `:w` (write) `:q` (quit).
+To quickly open helix config file `:config-open`, although you can also type `:co` and use `<tab>` to
+autocomplete.
+
+After you modify your config file, save it (`:w`) and use `:config-reload` to activate your changes
+without leaving Helix.
+
+[Tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) is an interractive mode you can
```

Comment:
Wow, so much good work! I like the direction you're going here, but moving the tutor to `:` section is not a strong move IMHO. I think it should be left at the top as it will be familiar to everyone who has a vim background (which is most I'd think).

Maybe have a `<space>?` guide quite early? It's a nice tool for users of all experience levels.